base/applications/notepad/lang/fr-FR.rc

```diff
@@ -75,35 +75,35 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_CAPTION | WS_SYSMENU | DS_CONTEXTHELP
 FONT 8, "MS Shell Dlg"
 CAPTION "Page Setup"
 BEGIN
-    GROUPBOX "Preview", 0, 240, 6, 120, 153, BS_GROUPBOX
+    GROUPBOX "Aperçu", 0, 240, 6, 120, 153, BS_GROUPBOX
     CONTROL "", rct1, "Static", SS_WHITERECT, 260, 42, 80, 80
     CONTROL "", rct2, "Static", SS_GRAYRECT, 340, 46, 4, 80
     CONTROL "", rct3, "Static", SS_GRAYRECT, 264, 122, 80, 4
-    GROUPBOX "Paper", grp2, 8, 6, 224, 56, BS_GROUPBOX
-    LTEXT "&Size:", stc2, 16, 22, 36, 8
+    GROUPBOX "Papier", grp2, 8, 6, 224, 56, BS_GROUPBOX
+    LTEXT "&Dimension :", stc2, 16, 22, 36, 8
```

Comment:
```suggestion
    LTEXT "&Taille :", stc2, 16, 22, 36, 8
```
to have something similar with the print dialog from comdlg32.

Reply:
Or "&Format" ?